source/dotnet/wholesale-api/Orchestrations/Extensions/DependencyInjection/WholesaleInboxExtensions.cs

```diff
@@ -43,10 +45,18 @@ public static IServiceCollection AddWholesaleInboxHandling(this IServiceCollecti
                 name: "WholesaleInboxQueue");
 
         services.AddWholesaleInboxHandler();
+
+        // EDI Wholesale inbox request handlers
         services.AddEdiModule(); // Edi module has Wholesale inbox handlers for requests from EDI
         services.AddCalculationsModule(configuration); // Calculations module is used by EDI
```

Comment:
```suggestion
        services.AddCalculationsModule(configuration); // Calculations module is used by EDI
```
If EDI is using this module, should it be registered by "AddEdiModule" ?

Reply:
Just a consideration

Reply:
That doesn't seem to be how it's done in Wholesale

Reply:
It doesn't seem to be how it's done in `Wholesale`, but it might just be because it is inconsistent. I've chosen to follow how the dependency injection was setup previously.